backends/tc/runtime/build-simple-p4

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
```

Comment:
This seems like a hardcoded script to set up an environment? Is this specific to a test? I would make it part of that tests folder if that is the case.

Reply:
Yes, it is to set up the specific environment we were using to test our examples
It's not specific to a test, but rather to all the new p4tc tests that would use the virtme environment
I can still move it to another directory though, if you think that would make more sense